internal/cmd/operator-sdk/cleanup/cmd.go

```diff
@@ -58,8 +54,12 @@ func NewCmd() *cobra.Command {
 			}
 		},
 	}
-	cmd.Flags().DurationVar(&timeout, "timeout", 2*time.Minute, "Time to wait for the command to complete before failing")
-	cfg.BindFlags(cmd.PersistentFlags())
```

Comment:
Where does the timeout flag come from now?

Reply:
It's configured in [`operator.Configuration`](https://github.com/estroz/operator-sdk/blob/feature/pod-secrets/internal/olm/operator/config.go#L63-L64) now.